<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyberLens ‚Äî Cyber Threat Analyzer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: #020617;
    color: #f8fafc;
    font-family: "Segoe UI", sans-serif;
}
#headerFrame {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
                stop:0 #1e293b, stop:1 #0f172a);
    border-top: 2px solid #38bdf8;
    border-bottom: 1px solid #334155;
    border-radius: 20px;
}
#iconLabel {
    border: 1px solid rgba(56, 189, 248, 0.4);
    border-radius: 10px;
    padding: 10px;
    font-size: 45px;
}
#brandBlock {
    border: 1px solid rgba(56, 189, 248, 0.4);
    border-radius: 10px;
    padding: 10px 25px;
    margin-left: 10px;
}
#titleLabel {
    color: #ffffff;
    font-size: 40px;
    font-weight: 900;
    letter-spacing: 6px;
}
#subtitleLabel {
    color: #38bdf8;
    font-size: 11px;
    font-weight: 800;
    letter-spacing: 3px;
    text-transform: uppercase;
    border-top: 1px solid rgba(255, 255, 255, 0.1);
    border-bottom: 1px solid rgba(255, 255, 255, 0.1);
    padding: 4px 0px;
    margin-top: 5px;
}
#modeLabel {
    color: #4ade80;
    background: rgba(74, 222, 128, 0.1);
    border: 1px solid #4ade80;
    border-radius: 15px;
    padding: 8px 20px;
    font-weight: 800;
}
#urlCard, #fileCard {
    background: #0f172a;
    border: 1px solid #1e293b;
    border-radius: 20px;
}
#cardTitle1, #cardTitle2 {
    color: #38bdf8;
    font-weight: 700;
    font-size: 15px;
}
QLineEdit {
    background: #020617;
    border: 2px solid #334155;
    border-radius: 12px;
    padding: 12px;
    color: #ffffff;
}
QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #6366f1, stop:1 #a855f7);
    color: #ffffff;
    border-radius: 12px;
    min-height: 50px;
    font-weight: 800;
}
#hashLabel, #urlRiskScore, #verdictLabel {
    background: #020617;
    border: 1px dashed #475569;
    border-radius: 12px;
    padding: 12px;
    color: #ffffff;
    font-weight: 600;
}
#historyBtn, #reportBtn {
    background: rgba(255, 255, 255, 0.05);
    border: 1px solid #334155;
    min-width: 240px;
    min-height: 48px;
    color: #ffffff;
    font-weight: 700;
    font-size: 14px;
}
#historyBtn:hover, #reportBtn:hover {
    background: rgba(56, 189, 248, 0.15);
    border: 1px solid #38bdf8;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing"><number>20</number></property>
    <property name="leftMargin"><number>40</number></property>
    <property name="topMargin"><number>30</number></property>
    <property name="rightMargin"><number>40</number></property>
    <property name="bottomMargin"><number>30</number></property>

    <!-- HEADER -->
    <item>
     <widget class="QFrame" name="headerFrame">
      <layout class="QHBoxLayout">
       <property name="spacing"><number>15</number></property>
       <property name="leftMargin"><number>20</number></property>
       <property name="topMargin"><number>15</number></property>
       <property name="rightMargin"><number>20</number></property>
       <property name="bottomMargin"><number>15</number></property>
       <item>
        <widget class="QLabel" name="iconLabel">
         <property name="text"><string>üõ°Ô∏è</string></property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="brandBlock">
         <layout class="QVBoxLayout" name="brandInnerLayout">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="text"><string>CYBERLENS</string></property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="subtitleLabel">
            <property name="text"><string>AI-POWERED THREAT DETECTION INTELLIGENCE</string></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="headerSpacer">
         <property name="orientation"><enum>Qt::Horizontal</enum></property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="modeLabel">
         <property name="text"><string>üåê ONLINE (VT &amp; GOOGLE SAFE)</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <!-- NETWORK CARD -->
    <item>
     <widget class="QFrame" name="urlCard">
      <layout class="QVBoxLayout">
       <property name="spacing"><number>15</number></property>
       <item>
        <widget class="QLabel" name="cardTitle1">
         <property name="text"><string>üåê NETWORK SECURITY</string></property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="urlInput">
         <property name="placeholderText"><string>Enter target URL...</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAnalyzeUrl">
         <property name="text"><string>üîç URL Analyzing</string></property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="urlRiskScore">
         <property name="text"><string>üì° Connection: Awaiting Scan...</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <!-- FILE CARD -->
    <item>
     <widget class="QFrame" name="fileCard">
      <layout class="QVBoxLayout">
       <property name="spacing"><number>15</number></property>
       <item>
        <widget class="QLabel" name="cardTitle2">
         <property name="text"><string>üìÅ FILE SYSTEM INTEGRITY</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowse">
         <property name="text"><string>üìÇ BROWSE SECURE STORAGE</string></property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing"><number>15</number></property>
         <item>
          <widget class="QLabel" name="hashLabel">
           <property name="text"><string>üîë SHA-256: --</string></property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="verdictLabel">
           <property name="text"><string>‚öñÔ∏è Verdict: Pending...</string></property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>

    <!-- BOTTOM BUTTONS -->
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing"><number>20</number></property>
      <item>
       <spacer name="leftSpacer">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="historyBtn">
        <property name="text"><string>üìú Scan History</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reportBtn">
        <property name="text"><string>üìä Generate Forensic Report</string></property>
       </widget>
      </item>
      <item>
       <spacer name="rightSpacer">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
       </spacer>
      </item>
     </layout>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
